app/models/katello/installed_package.rb

```diff
@@ -2,5 +2,7 @@ module Katello
   class InstalledPackage < Katello::Model
     has_many :hosts, :through => :host_installed_packages, :class_name => "::Host"
     has_many :host_installed_packages, :class_name => "Katello::HostInstalledPackage", :dependent => :destroy, :inverse_of => :installed_package
+
+    scoped_search :in => :host_installed_packages, :on => :name
```

Comment:
i get an error when i try to search for 'name = kernel'.  Removing the ':in => host_installed_packages" seems to fix it.